python/lsst/daf/butler/cli/opt/options.py

```diff
@@ -103,3 +103,12 @@ def makeCollectionType(context, param, value):
 verbose_option = MWOptionDecorator("-v", "--verbose",
                                    help="Increase verbosity.",
                                    is_flag=True)
+
+
+options_file_option = MWOptionDecorator("--options-file", "-@",
```

Comment:
of course this adds the `-@` short flag for butler as well. I don't mind either way, but I think it will work if you want to add it only in `pipetask`, where you use the `options_file_option` you can pass it there:
```
@click.command()
@options_file_option("-@")
def build(**kwargs):
    ...
```

Reply:
I want the short flag available everywhere. I realize that butler is not going to be a problem because the options are small but I didn't think it caused harm by having it be consistent.